moments/ViewController.swift

```diff
@@ -25,6 +25,7 @@ class ViewController: UIViewController, MKMapViewDelegate, CLLocationManagerDele
         let region: MKCoordinateRegion = MKCoordinateRegionMake(location, span)
         map.setRegion(region, animated: true)
         map.showsUserLocation = true
+        locationManager.stopUpdatingLocation()
```

Comment:
If you mean does the map stay unlocked while the user is moving (e.g., simulated ride), then yes! Essentially, the map loads centered on the user and then permits free movement (pan/zoom).